### PR TITLE
Add Ref to shared process namespaces

### PR DESCRIPTION
**Pull request by so0k**
The documentation talking about ways containers can talk to each other within a pod should include shared process namespace as a method and refer to the documentation talking about sharing process namespace

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight: 110
 {{% capture overview %}}
 
 This page shows how to use a Volume to communicate between two Containers running
-in the same Pod.
+in the same Pod. See also how to allow processes to communicate by [sharing process namespace](/docs/tasks/configure-pod-container/share-process-namespace/) between containers (Kubernetes server must be at or later than version v1.10).
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md]: Would it make sense to use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) here, to highlight the need for Kubernetes v1.10 or later?

**Comment by Rajakavitha1** [comment]
@so0k Thanks for your PR!!! Could you please incorporate the changes that @sftim !!!!

File: content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md]: (If using feature-states, it'll be important to show the reader what that applies to. The feature-state for sharing volumes is different from the feature state for the further reading about sharing processes between containers in a Pod).

Comment (so0k) [reply content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md]: indeed, the page linked already clarifies feature state, adding it to the reference which is in the volumes documentation would be confusing.

Can you please re-clarify what changes you want me to make (if any) @sftim ?

**Comment by so0k** [comment]
@Rajakavitha1  - done - please review and advise

File: content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 110
 {{% capture overview %}}
 
 This page shows how to use a Volume to communicate between two Containers running
-in the same Pod.
+in the same Pod. See also how to allow processes to communicate by [sharing process namespace](/docs/tasks/configure-pod-container/share-process-namespace/) between containers.
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md]: This extra text seems like it's a better fit for the `whatsnext` section than for the page overview.
How about keeping just the `whatsnext` changes?

**Pull request merged**